SampleWebView.Avalonia/SampleWebView.Avalonia.csproj

```diff
@@ -4,7 +4,7 @@
     <OutputType>WinExe</OutputType>
     <TargetFramework>net6.0</TargetFramework>
     <RollForward>LatestMajor</RollForward>
-    <Configurations>Debug;Release;ReleaseAvalonia;ReleaseWPF;ReleaseAvaloniaRemoteDebugSupport</Configurations>
```

Comment:
Rename ReleaseAvalonia -> to Release*

Reply:
done. Also renamed ReleaseAvaloniaRemoteDebugSupport to ReleaseRemoteDebugSupport